src/pages/Match/MatchPendingText.tsx

```diff
@@ -0,0 +1,16 @@
+import { CenteredText } from 'components/CenteredText';
+import { distanceInWordsToNow } from 'date-fns';
```

Comment:
Does this blow up bundle size because it can't tree-shake? 🤔It's pretty varied in the codebase so I'm not going to lose sleep over it.
```suggestion
import * as distanceInWordsToNow from 'date-fns/distance_in_words_to_now';
```

Reply:
yeah, I guess it's better to only import the individual modules. We already import all of date-fns elsewhere but I'll fix that too

Reply:
as an aside our entire bundle is ~200kB. BA is over 4MB!!!